features/webdriver-sensors.yml

```diff
@@ -0,0 +1,11 @@
+name: WebDriver Sensors
+description: WebDriver Sensors are WebDriver extension commands for creating, deleting, getting, and setting readings for virtual sensors (sensors that do not depend on underlying hardware or operating system support).
+spec: https://w3c.github.io/sensors/#automation
+group:
+  - sensors
+  - webdriver
+compat_features:
+  - webdriver.commands.CreateVirtualSensor
+  - webdriver.commands.DeleteVirtualSensor
+  - webdriver.commands.GetVirtualSensorInformation
+  - webdriver.commands.UpdateVirtualSensorReading
```

Comment:
I'm not certain this feature should standalone. It might be part of a more general generic sensors feature. Can we move this into drafts (i.e., `features/draft/webdriver-sensors.yml`) and reconsider it when someone (you, probably) gets to the stuff in `features/draft/spec/generic-sensor.yml`?